resources/articles/migration-service.md

```diff
@@ -27,6 +27,8 @@ response.
 
 ![Migration service](/images/migration-service.png)
 
+A sample migration service implementation can be found on GitHub: https://github.schibsted.io/spt-identity/fake-migration-service
```

Comment:
No, we decided making it public in the enterprise github was enough since all Schibsted companies can see it then. Provided they know that it's there of course (hence the link in the documentation). That was the idea anyway.